src/main/java/org/myrobotlab/document/transformer/Configuration.java

```diff
@@ -132,6 +137,12 @@ public List<String> getListParam(String name) {
     return null;
   }
 
+  public void setMapProperty(String name, Map map) {

```

Comment:
Would Map<String, String> be sufficient or is it expected to also contain other types?